src/exercise/06.js

```diff
@@ -18,11 +18,23 @@ function UsernameForm({onSubmitUsername}) {
 
   // 🐨 make sure to associate the label to the input.
   // to do so, set the value of 'htmlFor' prop of the label to the id of input
+  const [username, setUsername] = React.useState(null);
+
+  const handleSubmit = (event) => {
+    onSubmitUsername(username);
+
+    event.preventDefault();
+  };
+  const handleChange = (event) => {
```

Comment:
you could shorten if wanted :)
```suggestion
  const handleChange = (event) => setUsername(event.target.value.toLowerCase());
```